yt/loaders.py

```diff
@@ -754,6 +760,7 @@ def load_hexahedral_mesh(
     periodicity=(True, True, True),
     geometry="cartesian",
     unit_system="cgs",
+    use_mks_em_units=False,
```

Comment:
So this is what I meant to suggest early (re: keyword-only arguments)
```suggestion
    *,
    use_mks_em_units=False,
```